wagtail/coreutils.py

```diff
@@ -422,11 +422,12 @@ def get_dummy_request(*, path: str = "/", site: "Site" = None) -> HttpRequest:
     if site:
         server_name = site.hostname
         server_port = site.port
-    elif settings.ALLOWED_HOSTS == ["*"]:
```

Comment:
How about changing the original logic to use `in` instead?

```suggestion
    elif "*" in settings.ALLOWED_HOSTS
```

Reply:
Happy to make that change, but it would change the behaviour slightly so that `example.com` is used even if the first item in `ALLOWED_HOSTS` is not `*`.

Reply:
Yep, don't mind it, I've just discussed it in the other thread https://github.com/wagtail/wagtail/pull/11747#discussion_r1524494115 :)